xml/cl.xml

```diff
@@ -5496,6 +5537,18 @@ server's OpenCL/api-docs repository.
             <require comment="cl_device_info">
                 <enum name="CL_DEVICE_HALF_FP_CONFIG"/>
             </require>
+            <require comment="Constants">
+                <enum name="CL_HALF_DIG"/>
+                <enum name="CL_HALF_MANT_DIG"/>
+                <enum name="CL_HALF_MAX_10_EXP"/>
+                <enum name="CL_HALF_MAX_EXP"/>
+                <enum name="CL_HALF_MIN_10_EXP"/>
+                <enum name="CL_HALF_MIN_EXP"/>
+                <enum name="CL_HALF_RADIX"/>
+                <enum name="CL_HALF_MAX"/>
+                <enum name="CL_HALF_MIN"/>
+                <enum name="CL_HALF_EPSILON"/>
```

Comment:
These half constants are also defined in `cl_platform.h`.  Since these are `#defines` and not `typedefs` we could define them twice without a diagnostic (assuming they're defined to the same values, which hopefully they are!), though it might be better not to do this.

Do we want to:

1. Consider these constants to be parts of the API, unconditionally, also, by including them as part of "OpenCL 1.0" and not part of the `cl_khr_fp16` extension?
2. Take them out of `cl_platform.h` and define them in `cl_ext.h` instead, as part of the `cl_khr_fp16` extension?
3. Some other solution that involves special-casing these constants in the extension header generation scripts.

Note, I originally thought it was weird that `cl_half` was in "OpenCL 1.0", especially because `cl_double` is not, but perhaps this is intended and correct due to the `vload_half` and `vstore_half` built-in functions?